Source/CombatExtended/CombatExtended/Verbs/VerbPropertiesCE.cs

```diff
@@ -18,5 +18,6 @@ public class VerbPropertiesCE : VerbProperties
         public bool ejectsCasings = true;
         public bool ignorePartialLoSBlocker = false;
         public bool interruptibleBurst = true;
+        public List<ThingDef> cantShotWith = new List<ThingDef>();
```

Comment:
there's a typo there but maybe `disallowedProjectileDefs` would make the purpose clearer?

Reply:
Agree.
Fixed it